lib/LedgerSMB/Template.pm

```diff
@@ -213,7 +214,7 @@ sub new {
     $self->{debug} = $args{debug};
     $self->{binmode} = undef;
     $self->{outputfile} =
-        "${LedgerSMB::Sysconfig::tempdir}/$args{output_file}" if
+        "${LedgerSMB::Sysconfig::tempdir}/${LedgerSMB::Sysconfig::cache_template_subdir}/UI/$args{output_file}" if
```

Comment:
We may have to revise this then, because ${LedgerSMB::Sysconfig::tempdir}/$args{output_file} doesn't lead to a valid directory.